Makefile

```diff
@@ -0,0 +1,13 @@
+lint: node_modules
+	npm run lint 
+
+test: lint
+	npm test
+
+build: node_modules
```

Comment:
Weird...
Yup, just 'make' will install the dependencies.
then for test/lint [ and soon on CI ] : 
make lint
make test

But you right, now running 'make build' the same error appears here.
Please don't merge I will take a look on this now.
thanks!